### ISSUE TITLE
PDF Build

### ISSUE THREAD

**Issue by jawatson** [jawatson/pythonprop#2]
Installation fails after a make clean as the make files no longer include a rule for building the pdf.

**Comment by jawatson** [comment]
# Proposed Fix
1. Include make docs as a dependency of make dist
2. Remove the .pdf file from the make clean rule

**Comment by jawatson** [comment]
https://stackoverflow.com/questions/7027606/autoconf-automake-conditionals-and-dist-rules

**Comment by jaredd** [comment]
I ran into this issue. My fix was to place the MANPDF variable setting within the conditional, in docs/user/help/Makefile.am:

```
if HAVE_DBLATEX
MANPDF = $(DOC_MODULE).pdf
else
MANPDF = 
endif
```

**Comment by jawatson** [comment]
A pdf has been included in the cvs in v0.30.  This will only be rebuilt if the host system has dblatex.
